core/payment/src/lib.rs

```diff
@@ -23,7 +23,7 @@ pub mod migrations {
     struct _Dummy;
 }
 
-pub const DEFAULT_PAYMENT_PLATFORM: &str = "NGNT";
+pub const DEFAULT_PAYMENT_PLATFORM: &str = "ZK-NGNT";
```

Comment:
Is this necessary for testing? We cannot make it default *yet*.

Reply:
Updated, tested both scenario's and they are working :)